sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxGatewayStoreModel.java

```diff
@@ -530,6 +531,11 @@ public Flux<OpenConnectionResponse> openConnectionsAndInitCaches(CosmosContainer
         return Flux.empty();
     }
 
+    @Override
+    public void configFaultInjectionRule(IFaultInjectionRuleInternal rule) {
+        //To be implemented
```

Comment:
Should we throw NotSupportedException or similar?

Reply:
throw new NotImplementedException("configFaultInjectionRule is not supported in httpTransportClient");

Reply:
added throw